eoscostarica/eos-rate
# Delete user rate

### GH Issue

#445 #440 #463 

- Add new routes (About, Terms of Use. and Help)
- Add logic to scroll to top when view button proxy card is clicked
- Add mutation to delete user rate

### Steps to test

1. run make dev
2. go to `http://localhost:3000`
3. login and go to see sidebar menu (note 3 new routes)
4. go to Proxies list and click the card view button
5. go to account and try to delete a user rate

### Checklist
